docs/Deploying/Configuration.md

```diff
@@ -827,6 +906,16 @@ Valid time units are `s`, `m`, `h`.
   	duration: 4h
   ```
 
+##### Example YAML
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  session:
+  	duration: 4h
```

Comment:
@ksatter is the last tab for `duration` intentional. Asking because of the importance of space in YAML files